tests/sentry/snuba/test_discover.py

```diff
@@ -1468,18 +1468,23 @@ def test_failure_rate(self):
                 if expected_length > 1:
                     assert data[1]["failure_rate"] == 0.3, use_snql
 
-    def test_percentile(self):
-        project = self.create_project()
-
+    def _create_percentile_events(self, project):
         for i in range(6):
+            start = before_now(minutes=3)
+            end = start - timedelta(minutes=1 + i)
```

Comment:
not in this PR, but should we consider adding a util function that just gives stable start and end times with optional args like start_before_now, end_before_now, a timedelta etc

Reply:
yeahhh should've commented this, looking at the remaining flakes it literally all the same reason lol.
Going to add a param to load_data that allows you to set duration then start refactoring tests